docs/user/ppl/cmd/subquery.md

```
@@ -0,0 +1,197 @@
+source = outer | where a not in [ source = inner | fields b ]
+source = outer | where (a) not in [ source = inner | fields b ]
+source = outer | where (a,b,c) not in [ source = inner | fields d,e,f ]
+source = outer a in [ source = inner | fields b ] // search filtering with subquery
+source = outer a not in [ source = inner | fields b ] // search filtering with subquery)
+source = outer | where a in [ source = inner1 | where b not in [ source = inner2 | fields c ] | fields b ] // nested
+source = table1 | inner join left = l right = r on l.a = r.a AND r.a in [ source = inner | fields d ] | fields l.a, r.a, b, c //as join filter
+```
+  
+ExistsSubquery: 
+  
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)